### ISSUE TITLE
Excess characters appear when saving input after using Ctrl O

### ISSUE THREAD

**Issue by duguanyue** [asciimoo/wuzz#137]
Every time when I use Ctrl O to open an editor, modify some input, and then save it, this occurs. 

What I've saved should be `https://www.google.com/`, but after saving via `:wq`, it changes to `2;2R3;1R>0;95;0c10;rgb:d067/d2e8/d15111;rgb:1d1c/1f1f/2120https://www.google.com/`.

Seems to have added some color infomation.

I'm using macOS Mojave 10.14.6 + iTerm2 + oh_my_zsh.

![image](https://user-images.githubusercontent.com/17875441/110411885-46860400-80c6-11eb-8dc2-23915bf96cbb.png)

![image](https://user-images.githubusercontent.com/17875441/110411849-353cf780-80c6-11eb-952c-7ea29f14a45c.png)

**Comment by asciimoo** [comment]
Yeah, this is definitely a bug. I have no clue what causes it, any help would be appreciated.

**Comment by lewesche** [comment]
See potential fix in #138
